docs/contributor/pocs/otlp-logs.md

```diff
@@ -104,3 +107,63 @@ In the used configuration, we move the original log record to the **original** a
 The OpenTelemetry Collector setup is able to extract the log message from different attributes, depending on their presence. This means that it is possible to support different logging libraries.
 
 Non-JSON logs are preserved in the **body** field until the enrichment with resource attributes is completed.
+
+## Buffering and Backpressure
+
+### Scope and Goals
+
+After evaluating the filelog receiver configuration in [Log Record Parsing](#log-record-parsing), we want to test the buffering and backpressure capabilities of the OpenTelemetry Collector. The OpenTelemetry-based logging solution should give similar resilience and guarantees about log delivery as the current logging solution.
+
+## Setup for Buffering and Backpressure
+
+We split the OpenTelemetry Collector for log processing to an agent (DaemonSet) and a gateway (StatefulSet). The agent uses the same configuration as shown in [Log Record Parsing](#log-record-parsing) to read logs from the host file system, and converts them to the OTLP format, while the gateway adds Kubernetes metadata and ensures that no logs are lost if the backend fails.
+
+The following figure shows the different plugins that are configured in the processing pipeline. Note the use of the batch processor in the gateway, which introduces asynchronicity to the pipeline and causes that backpressure is not propagated back to the agent. To minimize the risk of log loss due to the batch processors properties, a persistent exporter queue was introduced in the gateway, which uses a persistent volume to buffer logs in case of a backend failure.
+
+![Otel Collector Setup](./assets/otlp-logs.drawio.svg)
```

Comment:
This diagram looks weird when rendered: 
<img width="1036" alt="image" src="https://github.com/kyma-project/telemetry-manager/assets/76950046/8900a033-6710-49bd-a5a8-e388437bdc4c">

Consider changing the colors and background; for example, according to our [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/09-diagrams.md).